source/extensions/filters/network/redis_proxy/router_impl.h

```diff
@@ -75,6 +75,12 @@ class PrefixRoutes : public Router {
   void setReadFilterCallback(Network::ReadFilterCallbacks* callbacks) override {
     callbacks_ = callbacks;
   }
+  /**
```

Comment:
nit: please add a new line before the comments

Reply:
done.